.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm # See documentation for possible values
+    directory: "/" # Location of package manifests
```

Comment:
Big assumption that lerna support is built in as it is supported for dependabot security updates so hopefully it just works here.  If not, we can add each package  individually to the yaml